tests/test_transforms/test_inference/test_transform.py

```diff
@@ -455,11 +459,6 @@ def _test_transform_train_new_segments(self, ts, transform, train_segments, expe
     @pytest.mark.parametrize(
         "transform, dataset_name, expected_changes",
         [
-            (
```

Comment:
???

Reply:
there were 2 same tests with AddConst in line 459 and 498. you can check it in master

Reply:
And what?

Reply:
It is not a good practice to change sth in the places, which are not connected to you issue